frontend/amundsen_application/static/.betterer.results

```diff
@@ -977,11 +987,11 @@ exports[`eslint`] = {
     "js/pages/TableDetailPage/index.spec.tsx:1320221888": [
       [64, 6, 25, "Use object destructuring.", "1230260048"]
     ],
-    "js/pages/TableDetailPage/index.tsx:2519576402": [
-      [137, 2, 20, "key should be placed after componentDidUpdate", "3916788587"],
-      [147, 22, 12, "\'getTableData\' is already declared in the upper scope.", "3938384029"],
-      [164, 6, 12, "\'getTableData\' is already declared in the upper scope.", "3938384029"],
-      [235, 6, 28, "\'openRequestDescriptionDialog\' is already declared in the upper scope.", "4102713454"]
+    "js/pages/TableDetailPage/index.tsx:902622043": [
+      [138, 2, 20, "key should be placed after componentDidUpdate", "3916788587"],
+      [148, 22, 12, "\'getTableData\' is already declared in the upper scope.", "3938384029"],
+      [165, 6, 12, "\'getTableData\' is already declared in the upper scope.", "3938384029"],
+      [236, 6, 28, "\'openRequestDescriptionDialog\' is already declared in the upper scope.", "4102713454"]
     ],
     "js/utils/textUtils.ts:2545492889": [
       [19, 6, 46, "Unexpected lexical declaration in case block.", "156477898"]
```

Comment:
Is this file autogenerated?

Reply:
yeah. It is auto generated with `npm run betterer`

Reply:
I just modified js/pages/TableDetailPage/DataFreshnessButton/index.tsx to make it compatible with eslint so we can still pass the frontend unit tests without any change in`.betterer.results`. It means that we will have less files to check when we want to upgrade Amundsen and cherry pick commits.